state_of_html.yml

```
@@ -91,36 +91,36 @@ translations:
   # forms
 
   - key: features.form_controls
-    t: Form Controls
+    t: Controles de Formulário
   - key: features.form_controls.question
-    t: Which form controls have you used?
+    t: Quais controles de formulário você já utilizou?
 
   - key: features.form_input_types
-    t: Input Types
+    t: Tipos de Input
   - key: features.form_input_types.question
-    t: Which input types have you used?
+    t: Quais tipos de input você já utilizou?
   - key: options.form_input_types.na
-    t: <span aria-hidden="true">🚫</span> None of the above
+    t: <span aria-hidden="true">🚫</span> Nenhuma das opções acima
 
   - key: features.form_validation_features
-    t: Form Validation
+    t: Validação de formulário
   - key: features.form_validation_features.question
-    t: Which of the following form validation features have you used?
+    t: Quais dos seguintes recursos de validação de formulário você já utilizou?
   - key: options.form_validation_features.na
-    t: <span aria-hidden="true">🚫</span> None of the above
+    t: <span aria-hidden="true">🚫</span> Nenhuma das opções acima
 
   - key: features.forms_pain_points
-    t: Form Pain Points
+    t: Pontos problemáticos do formulário
   - key: features.forms_pain_points.question
-    t: What are your pain points around HTML forms?
+    t: Quais são os seus pontos problemáticos com os formulários HTML?
   - key: features.forms_pain_points.description
     t: |
-      Think of the various form controls (e.g. `<input>`, `<textarea>`, `<select>`, `<button>` etc.),
-      behaviors (e.g. validation, autocomplete), handling forms via JS, editing APIs (e.g. `contentEditable`) etc.
-      Anything that bothers you is fair game — browser support, missing features, styling, etc.
+      Pense nos diversos controles de formulário (por exemplo `<input>`, `<textarea>`, `<select>`, `<button>` etc.),
+      comportamentos (por exemplo validação, autocomplete), tratamento de formulário com JS, edição de APIs (por exemplo `contentEditable`) etc.
```

Pra manter um padrão entre os idiomas, acho que podemos usar a abreviação de "exemplo".
```suggestion
      comportamentos (ex. validação, autocomplete), tratamento de formulário com JS, edição de APIs (ex. `contentEditable`) etc.
```

---

optei por `por exemplo` já que `e.g.` na tradução literal é sempre `por exemplo` mesmo

---

Creio eu que `por ex.` seria uma boa

---

É que e.g. também é uma abreviação (do latin, mas ainda assim uma abreviação), e neste caso a sugestão é traduzir também na abreviação pro português.

A versão em es-ES é um bom exemplo também de padrão:
https://github.com/Devographics/locale-es-ES/blob/main/state_of_html.yml#L120

---

Acho que `ex.` ou `p. ex.`, para manter abreviado.

https://dicionario.priberam.org/p.%20ex.

---

Concordo com o @romulo1984. Devemos manter, na verdade, o padrão que usamos aqui no Brasil, que é `p. ex.`. Vamos padronizar dessa forma, acredito que seja a mais legível e igualmente fiel ao original.

---

Corrigido para `p. ex.`